test/specs/lib/keyboardHandlerFilter-test.ts

```diff
@@ -35,7 +35,7 @@ const getEventArg = (
     target: null,
     timeStamp: null,
     type: null,
-  }
+  } as any
```

Comment:
@sophieH29 to take a look and how we can consolidate `KeyboardEvent` and `React. KeyboardEvent` interfaces